saml2/saml2-service-provider/src/test/java/org/springframework/security/saml2/provider/service/web/authentication/logout/HttpSessionLogoutRequestRepositoryTests.java

```diff
@@ -77,6 +85,21 @@ public void loadLogoutRequestWhenMultipleSavedThenReplacesLogoutRequest() {
 		assertThat(this.logoutRequestRepository.loadLogoutRequest(request)).isEqualTo(two);
 	}
 
+	@Test
+	void serializeAndDeserializeSaml2LogoutRequest(@TempDir Path tempDir) throws IOException, ClassNotFoundException {
```

Comment:
Instead of using the file system, will you please use `ByteArrayInputStream` and `ByteArrayOutputStream`?

Reply:
Agree, thats clean and efficient.